### PR TITLE
fix(wash): windows path to target

### LINKED ISSUE #1481: [BUG] wash build on windows appears to be broken
### Affected project(s)

- [ ] documentation
- [ ] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Describe the bug

wash build failing on windows

### Steps to reproduce

Following the quickstart on windows:
```bash
cargo install wash-cli
wash new actor --git wasmcloud/wasmcloud --subfolder examples/rust/actors/http-hello-world hello
rustup target add wasm32-wasi
cd hello
wash build
```

`wash build` fails with:
`Could not find compiled wasm file, please ensure \\?\C:\Users\marku\wcloudtraining\hello\target/wasm32-wasi/release/http_hello_world.wasm exists`

### Expected behavior

No errors and a signed component is built.

### Environment

- OS: [Windows]
- Shell [powershell]
- wasmcloud Version [0.81.0]

### Screenshots / Logs / Additional context

Potentially related https://github.com/wasmCloud/wash/issues/341

### PR DESCRIPTION
**Pull request by ricochet**
Verbatim paths on Windows are not well supported,
e.g. "\\\\?\\C:\\Users..." while technically valid, causes some fs api's like `exists` to fail errantly.

The fix is to use a third party lib normpath to normalize the path to the wasm
binary.

Related issue: https://github.com/rust-lang/cargo/issues/9770

Fixes #1481

Signed-off-by: Bailey Hayes <behayes2@gmail.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -118,6 +118,7 @@ names = { version = "0.14", default-features = false }
 nix = { version = "0.27", default-features = false }
 nkeys = { version = "0.3", default-features = false }
 notify = { version = "6", default-features = false }
+normpath = { version = "1.1.1" , default-features = false }
```
Comment (t3hmrman) [inline Cargo.toml:121]:
```suggestion
normpath = { version = "1.1" , default-features = false }
```

**Review by t3hmrman — APPROVED** [review]
LGTM :rocket: 

I'd love a test for this (especially if we could constraint it to only run on windows), but not going to hold up the fix getting out for that (could be a follow up ticket for later)

File: Cargo.toml (hunk 1 above)
Comment (vados-cosmonic) [inline Cargo.toml:121]:
```suggestion
normpath = { version = "1.1" , default-features = false }
```

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket: 

Would love a test for this so we can catch it in the future when we change/to make sure it keeps not being broken, but that can be a follow up thing!

**Review by brooksmtownsend — APPROVED** [review]
Just a note we'll need to make sure this makes it into the upcoming wash-lib release so we can release the fix in wash-cli

**Comment by connorsmith256** [comment]
Hmm, it looks like a lot of the `wash build` integration tests are failing

**Pull request merged**